router/server.go

```diff
@@ -102,6 +103,29 @@ func main() {
 	apiPort := flag.String("api-port", "", "api listen port")
 	flag.Parse()
 
+	if ports := os.Getenv("HTTP_PORTS"); ports != "" {
```

Comment:
Hrm. CLI confirmation would not be clean: it requires altering the `env` command. Maybe test for a "yes I'm sure" variable being set, and flunk (rolling back to prior settings) if it isn't set?